pkg/api/apis/operators/v1alpha2/operatorgroup_types.go

```diff
@@ -0,0 +1,33 @@
+package v1alpha2
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+type OperatorGroupSpec struct {
+	Selector       metav1.LabelSelector  `json:"selector"`
+	ServiceAccount corev1.ServiceAccount `json:"serviceAccount"`
```

Comment:
I think we want both of these to be `omittempty`

Reply:
Yep, and in absence of a serviceaccount we'd use the one that OLM is running as (system:controller:operator-lifecycle-manager)